lib/jsonapi/fetchdata/parameters/parsers/paginate.rb

```diff
@@ -8,7 +8,10 @@ class Paginate < Parser
 
           def parse params={}
             number, limit, size = params.values_at('number', 'limit', 'size')
-            { 'number': (number || 1).to_i, 'size': (size || limit || Kaminari.config.max_per_page || 40).to_i }
+            {
+              number: (number || 1).to_i,
+              size: (size || limit || Kaminari.config.max_per_page || 40).to_i
```

Comment:
The config can be a good start but the Kaminari gem would be tightly coupled this way, it will be hard to replace it with some other gem that does the same. I would prefer to add some kind of adapter where you can wrap the Kaminari gem in, then it would be easily replaceable by other adapters that have their own implementation for the pagination.